sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/CreateHiveTableAsSelectCommand.scala

```diff
@@ -56,34 +57,36 @@ case class CreateHiveTableAsSelectCommand(
         return Seq.empty
       }
 
-      sparkSession.sessionState.executePlan(
-        InsertIntoTable(
-          UnresolvedRelation(tableIdentifier),
-          Map(),
-          query,
-          overwrite = false,
-          ifPartitionNotExists = false)).toRdd
+      InsertIntoHiveTable(
```

Comment:
@cloud-fan this change from `InsertIntoTable` to `InsertIntoHiveTable` introduces a regression [SPARK-25271](https://issues.apache.org/jira/browse/SPARK-25271), I'd like to revert it back to use `InsertIntoTable`, WDYT?

Reply:
Will it affect web UI SQL tab?

Reply:
ah good catch! I don't think we can revert here, as we need to execute the physical plan given as a parameter.

I think we should improve the hive table conversion optimizer rule, and handle CTAS as well.

Reply:
Ok. I see. Thanks.